lib/run/summary.js

```diff
@@ -12,7 +12,15 @@ var _ = require('lodash'),
  */
 RunSummary = function RunSummary (emitter, options) {
     // keep a copy of this instance since, we need to refer to this from various events
-    var summary = this;
+    var summary = this,
+        skipExecutionsSummary = !_.get(options, 'executionsSummary', true);
+
+    if (!skipExecutionsSummary) {
+        // skipExecutionsSummary in case the option is specified, or only cli is specified as reporter
+        skipExecutionsSummary = (options && Array.isArray(options.reporters) &&
+            options.reporters.length === 1 &&
+            options.reporters.includes('cli'));
```

Comment:
Other than the callback method passed to newman run I can't find any other way to differentiate between the newman cli or library calls.
Is there any other way you can suggest to find the difference? I don't want to rely on some extra param passing for lib calls as it might break backward compatibility